pkg/skaffold/schema/defaults/defaults_test.go

```diff
@@ -341,6 +341,20 @@ func TestSetPortForwardLocalPort(t *testing.T) {
 	testutil.CheckDeepEqual(t, 9000, cfg.PortForward[1].LocalPort)
 }
 
+func TestSetPortForwardOnEmptyPortForwardResource(t *testing.T) {
+	cfg := &latest.SkaffoldConfig{
+		Pipeline: latest.Pipeline{
+			Build: latest.BuildConfig{},
+			PortForward: []*latest.PortForwardResource{
+				nil,
+			},
+		},
+	}
+	err := Set(cfg)
+	SetDefaultDeployer(cfg)
```

Comment:
does `SetDefaultDeployer()` have to be called for this test?

Reply:
Not at all. I removed this line.